README.md

```diff
@@ -4,6 +4,19 @@
 
 This repository contains Documentation, Release Notes and an Issue Tracker for the Renovate "Pro Edition" container.
 
+## Getting Started
```

Comment:
Terraform is definitely an optional requirement - we don't need to mandate it - so we should move this to somewhere else that doesn't imply it's the only way to set up Renovate Pro